docs/api.md

```diff
@@ -298,6 +298,72 @@ Create a new wallet.
 }
 ```
 
+### `POST /api/wallet/withdraw`
+
+Withdraw funds to the target account, using Master Safe first and
+falling back to Master EOA if needed.
+
+**Request Body:**
+
+```json
+{
+  "password": "your_password",
+  "to": "0x...",
+  "withdraw_assets": {
+    "gnosis": {
+      "0x0000000000000000000000000000000000000000": 1000000000000000000,
+      "0x...": 500000000000000000
```

Comment:
so we rely on FE to pass the amount that they want to withdraw. But what should they pass if they want to completely empty the master safe and EOA? They can't pass the exact balance because some dust is required for gas fees.

Reply:
I see two solutions:
1. The FE leave some hardcoded dust when the user wants to withdraw everything.
2. We support some flag to drain it completely.